external/openglcts/modules/gles2/es2cTexture3DTests.cpp

```diff
@@ -1051,6 +1051,12 @@ CompressedTexture3DCase::IterateResult CompressedTexture3DCase::iterate(void)
 		m_testCtx.setTestResult(QP_TEST_RESULT_NOT_SUPPORTED, "Compressed format not supported by implementation");
 		return STOP;
 	}
+	
+	if (!contextInfo.isExtensionSupported("GL_KHR_texture_compression_astc_hdr"))
```

Comment:
It looks like that with this change you will skip any compressed format not only those that are introduced by ASTC HDR. 

In my opinion modules/gles31/functional/es31fCopyImageTests.cpp:1892 . provides a better solutio. You should probably adapt it to your needs.

Reply:
ok, thank you ,i will try it

Reply:
> It looks like that with this change you will skip any compressed format not only those that are introduced by ASTC HDR.
> 
> In my opinion modules/gles31/functional/es31fCopyImageTests.cpp:1892 . provides a better solutio. You should probably adapt it to your needs.

please review again , thank you